packages/components/ng-ui-router-layout/package.json

```diff
@@ -43,6 +43,7 @@
   "peerDependencies": {
     "@uirouter/angularjs": "^1.0.22",
     "angular": "^1.5.0",
-    "angular-ui-bootstrap": "~1.3.3"
+    "angular-ui-bootstrap": "~1.3.3",
+    "ovh-ui-angular": "^3.8.2"
```

Comment:
You'll need `ovh-ui-kit` as peerDep also to be sure the styles are imported I think

Reply:
done, thx